Packs/Netmiko/Integrations/Netmiko/Netmiko.py

```diff
@@ -0,0 +1,183 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' IMPORTS '''
+
+
+import io
+import json
+import re
+import sys
+from datetime import datetime
+from time import sleep
+
+import paramiko
+from netmiko import Netmiko
+from netmiko.ssh_dispatcher import platforms
+
+''' HELPER FUNCTIONS '''
+
+
+def myfakefile(keys):
+    myfakefile.readlines = lambda: keys.split("\n")
+    return myfakefile
+
+
+class Client:
+    def __init__(self, platform, hostname, username, password, port, keys):
+        self.platform = platform
+        self.hostname = hostname
+        self.username = username
+        self.password = password
+        self.port = port
+        self.keys = keys
+        self.net_connect = None
+
+    def connect(self):
+        if self.keys:
+            try:
+                self.net_connect = Netmiko(device_type=self.platform, host=self.hostname, port=self.port,
+                                           pkey=self.keys, use_keys=True, username=self.username, passphrase=self.password)
+            except Exception as err:
+                return_error(err)
+        else:
+            try:
+                self.net_connect = Netmiko(device_type=self.platform, host=self.hostname, port=self.port,
+                                           use_keys=True, username=self.username, password=self.password)
+            except Exception as err:
+                return_error(err)
+
+    def disconnect(self):
+        try:
+            if self.net_connect:
+                self.net_connect.disconnect()
+        except Exception as err:
+            return_error(err)
+
+    def cmds(self, exitRequired, exitChar, commands, enable, isConfig):
+        try:
+            output = {"Hostname": self.hostname, "Platform": self.platform, "Commands": []}
+            self.connect()
+            sleep(2)
```

Comment:
Unfortunately, we do not allow `sleep` in our code, please change it.

Reply:
@ChanochShayner Changed.